index.html

```diff
@@ -16,11 +25,11 @@
     {% if announceyear > nowyear or announceday >= nowday and announceyear >= nowyear %}
       <div class ="announcement">
         {{announcement.output}}
-      </div> <!-- /.announcement -->
+      </div>
     {% endif %}
   {% endfor %}
 
-</section> <!-- /.intro-header -->
+</section>  -->
```

Comment:
Was not sure how to include it in the homepage .. maybe we can move it to its own page?